openedx/features/job_board/models.py

```diff
@@ -1,31 +1,49 @@
-    responsibilities = models.TextField(blank=True, null=True)
-    website_link = models.URLField(max_length=255, blank=True, null=True)
-    contact_email = models.EmailField(max_length=255)
-    logo = models.ImageField(upload_to='job-board/uploaded-logos/', blank=True, null=True)
+    title = models.CharField(max_length=255, verbose_name='Job Title')
+    company = models.CharField(max_length=255, verbose_name='Organization Name')
+    type = models.CharField(max_length=255, choices=JOB_TYPE_CHOICES, blank=False, verbose_name='Job Type',
+                            default=0, help_text='Please select whether the job is onsite or can be done remotely.')
+    compensation = models.CharField(max_length=255, choices=JOB_COMPENSATION_CHOICES, verbose_name='Compensation',
+                                    blank=False, default=0, help_text='Please select the type of compensation you are '
+                                                                      'offering for this job.')
+    hours = models.CharField(max_length=255, choices=JOB_HOURS_CHOICES, verbose_name='Job Hours',
+                             blank=False, default=0, help_text='Please select the expected number of working hours '
+                                                               'required for this job.')
+    city = models.CharField(max_length=255, verbose_name='City')
+    country = CountryField(verbose_name='Country')
+    description = models.TextField(verbose_name='Job Description', help_text='Please share a brief description of the '
+                                                                             'job.')
+    function = models.TextField(blank=True, null=True, verbose_name='Job Function',
+                                help_text='Please share details about the expected functions associated with the job.')
+    responsibilities = models.TextField(blank=True, null=True, verbose_name='Job Responsibilities',
+                                        help_text='Please share the responsibilities associated with the job.')
+    website_link = models.URLField(max_length=255, blank=True, null=True, verbose_name='Website Link')
+    application_link = models.URLField(max_length=255, blank=True, null=True, verbose_name='Application Link',
+                                       help_text='Please share a link to the job application')
+    contact_email = models.EmailField(max_length=255, help_text='Please share a contact email for this job.',
+                                      verbose_name='Contact Email')
+    logo = models.ImageField(upload_to='job-board/uploaded-logos/', blank=True, null=True, verbose_name='Company Logo',
+                             validators=[
+                                 FileExtensionValidator(LOGO_ALLOWED_EXTENSION),
+                                 validate_file_size
+                             ],
+                             help_text='Please upload a file with your company\'s logo. (maximum 2MB)')
```

Comment:
Use LOGO_IMAGE_MAX_SIZE here, so that if anyone change constant value, it also reflect this help text automatically

Reply:
done